pkg/rpc/client/rpc.go

```diff
@@ -404,6 +404,38 @@ func (c *Client) GetUnspents(address string) (*result.Unspents, error) {
 	return resp, nil
 }
 
+// GetUTXOTransfers is a wrapper for getutxoransfers RPC. Address parameter
+// is mandatory, while all the others are optional. It's only supported since
+// since neo-go 0.77.0 with limit and page parameters only since neo-go 0.78.0.
+// These parameters are positional in the JSON-RPC call, you can't specify limit
+// and not specify start/stop for example.
+func (c *Client) GetUTXOTransfers(address string, start, stop *uint32, limit, page *int) (*result.GetUTXO, error) {
```

Comment:
What about making parameters `int64` and using `< 0` for the default value? With pointers it is hard to provide values as literals.

Reply:
Not sure I like `f(addr, -1, -1, -1, -1)`, though it's a valid concern.